hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/SCMSecurityProtocolServer.java

```diff
@@ -232,6 +233,16 @@ private void validateSecretKeyStatus() throws SCMSecretKeyException {
     }
   }
 
+  @Override
+  public boolean checkAndRotate(boolean force) throws TimeoutException {
+    try {
+      validateSecretKeyStatus();
+    } catch (SCMSecretKeyException e) {
+      LOGGER.error("Error while validating the secret key status.", e);
```

Comment:
Don't catch these validation exceptions on server side. We should let the translator to translate them to error code and transfer to client side.